common/src/main/java/com/wynntils/models/damage/DamageModel.java

```diff
@@ -60,6 +71,37 @@ public void onLabelIdentified(LabelIdentifiedEvent event) {
         lastDamageDealtTimestamp = System.currentTimeMillis();
     }
 
+    @SubscribeEvent
+    public void onWorldStateChange(WorldStateEvent event) {
+        areaDamageSet.clear();
+        focusedMobName = "";
+        focusedMobElementals = "";
+        focusedMobHealth = 0;
+        focusedMobHealthPercent = CappedValue.EMPTY;
+        focusedMobExpiryTime = -1L;
+        lastDamageDealtTimestamp = 0L;
+    }
+
+    public String getFocusedMobName() {
+        checkFocusedMobValidity();
+        return focusedMobName;
+    }
+
+    public String getFocusedMobElementals() {
+        checkFocusedMobValidity();
+        return focusedMobElementals; // TODO: Parse this into specific elements and expose as functions
```

Comment:
We usually don't do same-line TODOS, but really just a nit.

```suggestion
        // TODO: Parse this into specific elements and expose as functions
        return focusedMobElementals;
```